frontend/console/src/api/modules/use-stream-modules.ts

```diff
@@ -24,11 +24,11 @@ export const useStreamModules = () => {
           hasModules = true
           const newModuleNames = response.modules.map((m) => m.name)
           queryClient.setQueryData<Module[]>(queryKey, (prev = []) => {
-            return [...response.modules, ...prev.filter((m) => !newModuleNames.includes(m.name))]
+            return [...response.modules, ...prev.filter((m) => !newModuleNames.includes(m.name))].sort((a, b) => a.name.localeCompare(b.name))
           })
         }
       }
-      return hasModules ? queryClient.getQueryData(queryKey) : []
+      return hasModules ? (queryClient.getQueryData(queryKey) as Module[]) : []
```

Comment:
Because of the way the way deployments work now, it's possible to change to a new deployment while the other deployment is being shut down which will result in 2 deployments for the same module being active for a few seconds.

You can see this in the UI here:
![Screenshot 2024-10-10 at 9 23 02 AM](https://github.com/user-attachments/assets/b0e8f7c8-09d5-41a2-af87-de9d33d9f39d)

That's what this little bit of code in the `use-schema` is for: https://github.com/TBD54566975/ftl/blob/main/frontend/console/src/api/schema/use-schema.ts#L24-L43

Reply:
Ahhhh thanks for catching that! I'll do it in this PR so it can all go in cleanly.

Reply:
Done! I confirmed the module schema updates nicely upon redeploy too.